app.js

```diff
@@ -26,19 +24,28 @@ const { scheduleReminders, manuallyTriggerScheduledJobs } = require('./helpers/s
 // === Initialization/Configuration ===
 // ====================================
 
-// Create our own instance of the ExpressReceiver, which initializes an Express app
-const expressReceiver = new ExpressReceiver({
-  signingSecret: process.env.SLACK_SIGNING_SECRET
-})
-const expressApp = expressReceiver.app
-
-// Initialize slackInstaller OAuth helper from @slack/oauth package
-// (refer to https://github.com/slackapi/node-slack-sdk/tree/master/packages/oauth)
-const slackInstaller = new InstallProvider({
+// Initialize the Bolt app, including OAuth installer helpers
+// (refer to https://github.com/slackapi/bolt)
+const app = new App({
+  signingSecret: process.env.SLACK_SIGNING_SECRET,
   clientId: process.env.SLACK_CLIENT_ID,
   clientSecret: process.env.SLACK_CLIENT_SECRET,
-  authVersion: 'v2',
   stateSecret: randomStringGenerator(),
+  scopes: process.env.SLACK_BOT_SCOPES,
+  installerOptions: {
+    authVersion: 'v2',
```

Comment:
You don't need `authVersion`, `installPath`, `redirectUriPath` as the values you are specifying are the default values in bolt. 

Your `success` and `failure` are also very similar to the default version. Might just want to remove those and use the built in ones

Reply:
Ah thanks! All good points but for this example I'm going to keep this a little more verbose than necessary as an example so people can see how they could tweak the values without reading the docs :)